propagators/jaeger/jaeger_data_test.go

```diff
@@ -0,0 +1,269 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger_test
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel/api/trace"
+)
+
+const (
+	traceID16Str = "0000000000000000a3ce929d0e0e4736"
+	traceID32Str = "0000000000000000a3ce929d0e0e4736"
```

Comment:
Should these be different?  I'd expect the 128-bit ID to not be zero-padded.  What is the significance of the `16` and `32` in the variable names?  I'm having trouble connecting those to anything other than number of nibbles.

Reply:
Sorry should have removed the leading zero in `traceID16Str`. Fixed.